### ISSUE TITLE
Using Node 4 Features

### ISSUE THREAD

**Issue by hzoo** [babel/babel-eslint#397]
Since we dropped node < 4,

an easy win would be to change our tests to use template strings instead of

``` js
["asdfads",
"asdf"
].join('\n')
```

This doesn't have to be done in a single PR, and would be good for new contributors

**Comment by vgvenkat** [comment]
Hello there! Would this be conversion of all strings which use string concat using "+" in the [3 js test files](https://github.com/babel/babel-eslint/tree/master/test) as well or just the ones with ```join``` to use template strings?

**Comment by hzoo** [comment]
@vgvenkat Sorry there's already a pr for that particular change https://github.com/babel/babel-eslint/pull/410 (should of put a notice)

**Comment by vgvenkat** [comment]
oh. ok cool 👍🏻

**Comment by kaicataldo** [comment]
We did the same thing in the ESLint repo recently (there's one PR open that adds the final thing we wanted). This list might be helpful: https://github.com/eslint/eslint/issues/6407

**Comment by kaicataldo** [comment]
ESLint itself also has some fixers that could be used - `prefer-template` was just added recently and I used it on the main ESLint repo with no issues!

**Comment by karanjthakkar** [comment]
@hzoo Since #410 is merged, is there any other change that this issue is looking to address? I'd love to help out with other changes, if any, apart from the conversion to template strings.

**Comment by kaicataldo** [comment]
There actually is a PR open with some of this work done: https://github.com/babel/babel-eslint/pull/425

**Comment by hzoo** [comment]
Going to close as some of this is done and don't need to track
